src/sinopiaServer.js

```diff
@@ -50,11 +52,10 @@ export const resourceTemplateId2Json = [
   {id: 'resourceTemplate:bf2:Item:Chronology', json: chronologyRt}
 ]
 
+const emptyTemplate = { propertyTemplates : [{}] }
```

Comment:
Declare this once above the functions defined below, instead of defining it identically in each function.